### PR TITLE
added hello component

### PR DESCRIPTION
**Pull request by rashabaig**

### CODE REVIEW

**Comment by rashabaig** [comment]
What's the point of this code?

File: src/Hello.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class Hello extends Component {
+	render() {
+		return <div>{`Hello ${this.props.name}`}</div>;
```
Comment (gabriella1177) [inline src/Hello.js:5]: Is this the best to do it?

File: src/App.js [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
-				</a>
-			</header>
-		</div>
-	);
+import React, { Component } from 'react';
+import Hello from '../src/Hello';
+
+class App extends Component {
+	constructor() {
+		super();
```
Comment (rashabaig) [inline src/App.js:9]: States rule

File: src/Hello.js (hunk 1 above)
Comment (rashabaig) [reply src/Hello.js:5]: Yes

**Pull request merged**